schema/api-description.yaml

```diff
@@ -25,25 +25,20 @@ definitions:
   resource:
     type: object
     required:
-      - path
+      - uriTemplate
       - methods
     properties:
-      path:
+      uriTemplate:
```

Comment:
Just call this `template`. There's no ambiguity and `uriTemplate` isn't quite right anyway, since you need a URL (not a URI) to make an HTTP request.